src/graphql/entities/account.detailed/account.detailed.input.ts

```diff
@@ -17,6 +19,19 @@ export class GetAccountDetailedInput {
   }
 }
 
+@InputType({ description: "Input to retrieve the given from and size for." })
+export class GetFromAndSizeInput {
```

Comment:
wouldn't a name without "Get" be more suitable?

Reply:
Get prefix is applied to all graphql input classes